index.html

```diff
@@ -105,7 +105,8 @@ <h2 id="document_purpose">Purpose of this document</h2>
   Note that the terminology is based on that defined in [[JLREQ]]. </p>
 
 <aside class="note" title="Ruby and interlinear notes" id="n20200529001"> Interlinear notes are sometimes placed between lines
-      in the same position as ruby annotations.
+      in the same position as ruby annotations
+      (see also JLReq <a href="https://www.w3.org/TR/jlreq/#fig3_2_8">Figure 241: An example of a note in inter lines</a> for examples).
       These inter-linear notes are not covered in this document.
```

Comment:
Inconsistent use of inter-linear vs interlinear.  I would use interlinear (one word) (see https://www.google.com/search?client=firefox-b-d&q=define+interlinear)

Reply:
let me file separated issues (to here and JLReq) for performing check over the entire document.